core/src/main/resources/hudson/views/GlobalDefaultViewConfiguration/config.jelly

```diff
@@ -0,0 +1,36 @@
+<!--
+The MIT License
+Copyright (c) 2025, Markus Winter
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+  <j:if test="${app.views.size() > 0}">
+    <f:entry title="${%Default View}" field="defaultView">
+      <div class="jenkins-select">
+        <select name="primaryView" class="jenkins-select__input" disabled="${readOnlyMode ? 'true' : null}">
```

Comment:
Where is `disabled` coming from, is this new? Is this a change in behavior?

Reply:
It's basically fixing a bug that when in readonly mode (System Read permission only), the select was still working and not disabled. Not sure if it's worth to mark this as a bug